modules/cudf/src/groupby/base.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2020-2021, NVIDIA CORPORATION.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Series} from '@nvidia/cudf';
```

Comment:
```suggestion
import {Series} from '../series';
```

Reply:
I was actually meaning to start a discussion. There seems to be different usage all over and this is something I'd like to make a PR to clean up in general. But what is the policy here? Some library authors like to always use the same public APIs that users will as an extra assurance that public APis in fact function

Reply:
Not really any policy, it's _imperative_ we use local imports here, otherwise it breaks in the node repl.

Reply:
However in the tests we _should_ import public APIs using the module name, because `jest` and `ts-jest` do preprocessing to map[[1]](https://github.com/rapidsai/node-rapids/blob/main/modules/cudf/jest.config.js#L56)[[2]](https://github.com/rapidsai/node-rapids/blob/main/modules/cudf/test/tsconfig.json#L21-L22) the module names to the files in the source directory.